### DISCUSSION TITLE
Is it possible to mine a coin when the soft permanently diconnects to node?

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#4770] [General]
Is it possible to mine a coin when the soft permanently diconnects to node?
This week my plots were dead at leasr 3 days due to '
"Not Synced".I think a lot of people are left with created but dead plots.

**Comment by chiabug** [comment]
The plots are just a bunch of encrypted numbers sort of like having a bunch of lottery tickets in a basket.  This basket of numbers never changes and needs no network connection to remain valid.  The network periodically issues "challenges".  Your farming machine checks these "challenges" against the lottery tickets in your basket.  If one or more of your tickets is a match to the challenge hash then you POSSIBLY have a winning ticket.  Next your machine constructs a more detailed set of data called a "proof".  This proof is submitted back to the network and verified by peers for validity.  If you are the first machine to submit a  valid proof you stand a good chance of winning the block (and the reward).  Be aware -- submitting the first valid proof is not a guarantee that you will win the block.

Part of the "proof" data is based upon your machines knowledge of the current blockchain.  If you are not fully synced then you really have a stale copy of the blockchain and your proof will not be accepted by the network.  

Short answer -- without a fully synced node you will not win any block rewards.
